app/schema/questionnaire.py

```diff
@@ -43,16 +43,22 @@ def register(self, item):
             raise QuestionnaireException('{} is a duplicate id'.format(item.id))
 
         self.items_by_id[item.id] = item
-        self._register_alias(item)
 
         # Build the two-way relationship
         item.questionnaire = self
 
+    def register_aliases(self):
+        for _, item in self.items_by_id.items():
```

Comment:
Minor thing, but if the key is unused, then we don't need to call .items() to return key, value pairs.